### ISSUE TITLE
Accessibility: Audit Stats

### ISSUE THREAD

**Issue by khaykov** [wordpress-mobile/WordPress-Android#7287]
Audit Stats to make sure that all the functionality is accessible with TalkBack and Switch Access.

Part of the stats - GraphView is a custom view, that utilizes canvas and custom click logic. We need to make sure it's accessible.

- All the clickable elements should be marked as so.
- All nonactionable and decorative elements should be non-focusable.
- Relevant content should be grouped together.
- All important elements should be described through content description.

**Comment by rachelmcr** [comment]
Closing as resolved by https://github.com/wordpress-mobile/WordPress-Android/pull/7519.
